tasks/payara.yml

```diff
@@ -136,16 +133,7 @@
     mode: '0750'
     state: directory
 
-- name: workaround for FISH-7665
-  lineinfile:
-    path: '{{ payara_dir }}/glassfish/config/asenv.conf'
-    line: '{{ item }}'
-  with_items:
-    - 'export JAVA_TOOL_OPTIONS'
-    - 'JAVA_TOOL_OPTIONS="-Djdk.util.zip.disableZip64ExtraFieldValidation=true --add-opens=java.base/java.io=ALL-UNNAMED"'
```

Comment:
Huh. Is this safe to remove? We still have it in our Dockerfile:

```
$ grep UNN modules/container-base/src/main/docker/Dockerfile -B1
    # Workaround for FISH-7722: Failed to deploy war with @Stateless https://github.com/payara/Payara/issues/6337
    ${ASADMIN} create-jvm-options --add-opens=java.base/java.io=ALL-UNNAMED
```

Reply:
you're right, I thought this was fixed, but not until 6.2023.9 - will keep.